base/setup/usetup/usetup.c

```diff
@@ -3367,7 +3367,7 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
         if (!NT_SUCCESS(Status))
         {
             DPRINT1("BuildInstallPaths() failed. Status code: 0x%lx", Status);
-            PopupError("Failed to build the installation paths for the ReactOS installation directory!",
+            PopupError(MUIGetString(ERROR_NO_BUILD_PATH),
                        MUIGetString(STRING_CONTINUE),
```

Comment:
In this commit, you add "ENTER = Reboot computer" to this message (now stored in `ERROR_NO_BUILD_PATH`) , so `STRING_CONTINUE` ("ENTER = Continue") should be removed.